recipe/index.html

```diff
@@ -0,0 +1,86 @@
+            <h3>Ingredients</h3>
+            <ul>
+                    <li>1 tablespoon olive oil</li>
+                    <li>8 ounces boneless, skinless chicken thighs, cut into 1-inch chunks</li>
+                    <li>Kosher salt and freshly ground black pepper</li>
+                    <li>2 tablespoons unsalted butter</li>
+                    <li>3 cloves garlic, minced</li>
+                    <li>8 ounces cremini mushrooms, thinly sliced</li>
+                    <li>1 onion, diced</li>
+                    <li>3 carrots, peeled and diced</li>
+                    <li>2 stalks celery, diced</li>
+                    <li>1/2 teaspoon dried thyme</li>
+                    <li>1/4 cup all-purpose flour</li>
+                    <li>4 cups chicken stock</li>
+                    <li>1 bay leaf</li>
+                    <li>1/2 cup half and half, or more, as needed*</li>
+                    <li>2 tablespoons chopped fresh parsley       leaves</li>
+                    <li>1 sprig rosemary</li>
+            </ul>
+        </div>
+        <div class="direction">
+            <h3>Directions:</h3>
+            <ol>
+                <li>Heat olive oil in a large stockpot or Dutch oven over medium heat. Season chicken thighs with salt and pepper, to taste. Add chicken to the stockpot and cook until golden, about 2-3 minutes; set aside.</li>
+<li>Melt butter in the stockpot or Dutch oven over medium heat. Add garlic, mushrooms, onion, carrots and celery. Cook, stirring occasionally, until tender, about 3-4 minutes. Stir in thyme until fragrant, about 1 minute.</li>
+<li>Whisk in flour until lightly browned, about 1 minute. Whisk in chicken stock, bay leaf and chicken thighs, and cook, whisking constantly, until slightly thickened, about 4-5 minutes.</li>
+<li>Stir in half and half until heated through, about 1-2 minutes; season with salt and pepper, to taste. If the soup is too thick, add more half and half as needed until desired consistency is reached.</li>
+    <li>and rosemary, if desired.</li>
+            </ol>
+
+        </div>
+
+    </div>
+
+   <figure>
+    <div class="media">
+    <div id="media2">
+     <img src="https://hips.hearstapps.com/del.h-cdn.co/assets/17/39/1600x1066/gallery-1506695998-delish-creamy-chicken-rice.jpg?resize=980:*" width="600" height="300"></img>
+     <br>
+     <p>Photo credit:</p>
+<a href="https://damndelicious.net/2015/01/10/creamy-chicken-mushroom-soup/" onclick="showAlert()"><button class="button1">Try it Out!</button></a>
+  </div>
+</div>
+ </figure>
+</div>
+   
+<div>
+  
+  <div class = "Links"> <h2>Recomended Chicken and Mushroom Soup Recipes to Try:</h2> <p> <a href=" https://www.spendwithpennies.com/chicken-and-mushroom-soup/" onclick="showAlert()"><button class="button2">Spend with Pennies</button></a> </p> <p> <a href=" https://www.campbells.com/recipes/tasty-2-step-chicken-bake/" onclick="showAlert()"><button class="button3">Recipe from Cambell</button></a> </p> </div>
```

Comment:
Please put the children of the `<div>` here on newlines.

Reply:
Hi, I fixed it

Reply:
Thank you